### PR TITLE
HDDS-7294. Move ozone manager background services to a separate package

### PR DESCRIPTION
**Pull request by myskov**
## What changes were proposed in this pull request?

Ozone manager has four background services, and they share package 'org.apache.hadoop.ozone.om' with over 40 classes. It would be more clear to have background services in a separate package.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-7294

## How was this patch tested?

unit tests

### CODE REVIEW

**Comment by myskov** [comment]
@adoroszlai could you trigger verifications, please? BTW do you know why verifications didn't start automatically? I'm not a first-time contributor.

**Comment by adoroszlai** [comment]
> BTW do you know why verifications didn't start automatically? I'm not a first-time contributor.

Must be a github bug, I noticed this for others as well.

**Review by kaijchen — APPROVED** [review]
LGTM, thanks @myskov for the improvement.

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/package-info.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * <p>
+ * Utility classes to encode/decode DTO objects to/from byte array.
+ */
+
+/**
+ * The package contains Ozone Manager background services.
+ */
+package org.apache.hadoop.ozone.om.service;
```
Comment (kaijchen) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/package-info.java:24]: Missing newline at EOF here.

```suggestion
package org.apache.hadoop.ozone.om.service;

```

**Pull request merged**